### ISSUE TITLE
influx v0.9rc32 - version is not correcty displayed

### ISSUE THREAD

**Issue by dtouzeau** [influxdb/influxdb#2811]
Updated the latest rc32 available on download page for debian 
in rc31
```
/opt/influxdb/influxd version
[srvr] 2015/06/07 13:24:00 InfluxDB v0.9.0-rc31 (git: 8ade485b5865e642b128a7cc6eb20c9da73d1b48)
```

In rc32
```
/opt/influxdb/influxd version
InfluxDB v0.9 (git: unknown)
```
For the restore command ( also backup command is the same)

In rc31:
```
/opt/influxdb/influxd restore
2015/06/07 13:27:40 influxdb restore, version 0.9.0-rc31, commit 8ade485b5865e642b128a7cc6eb20c9da73d1b48
[srvr] 2015/06/07 13:27:40 No config provided, using default settings
[srvr] 2015/06/07 13:27:40 restore: snapshot path required
```
In rc32
```
/opt/influxdb/influxd restore
unknown command "restore"
Run 'influxd help' for usage
```

**Comment by toddboom** [comment]
@dtouzeau backups will be re-enabled in the next release. you can track #2812 for the status of that. i've renamed this issue to track the missing version info.

**Comment by toddboom** [comment]
@dtouzeau This is fixed with #2842 and will be included in RC33 - thanks!
